src/app/screens/Send/index.tsx

```diff
@@ -41,6 +41,14 @@ function Send() {
 
       if (lnurl) {
         const lnurlDetails = await lnurlLib.getDetails(lnurl);
+        if (!("tag" in lnurlDetails)) {
+          if (lnurlDetails.reason) {
+            toast.error(lnurlDetails.reason);
+          } else {
+            toast.error("Connection problem or invalid lnurl");
```

Comment:
wouldn't connection problems throw in `getDetails` ? so those would not end up here in this toast

Reply:
actually the error message is wrong, isn't it? 

this is a successful response but an invalid lnurl server response. 

so it is not a connection problem and not an invalid lnurl.